sensor_msgs/Image virtual_camera_image_bgr
sensor_msgs/Image virtual_camera_image_depth
uint8 line_type
---
float32[] embeddings
